app/Authentication/src/Http/Responses/FailedPasswordResetResponse.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Authentication\Http\Responses;
+
+use Illuminate\Contracts\Support\Responsable;
+use Illuminate\Http\JsonResponse;
+
+class FailedPasswordResetResponse implements Responsable
+{
+    /**
+    * The response status language key.
+    *
+    * @var string
+    */
+    protected $status;
+
+    /**
+     * Create a new response instance.
+     *
+     * @param  string  $status
+     * @return void
+     */
+    public function __construct(string $status)
+    {
+        $this->status = $status;
+    }
+
+    /**
+     * Create an HTTP response that represents the object.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function toResponse($request)
+    {
+        return $request->wantsJson()
+                    ? new JsonResponse(['message' => trans($this->status)], 200)
+                    : redirect()->route('login')->with('status', trans($this->status));
```

Comment:
This is not needed since it's an API only project. The line above can be the returned result.